zcash_client_backend/src/data_api/mem_wallet.rs

```diff
@@ -379,12 +381,76 @@ impl WalletWrite for MemoryWalletDb {
         todo!()
     }
 
+    /// Adds a sequence of blocks to the data store.
+    ///
+    /// Assumes blocks will be here in order.
     fn put_blocks(
         &mut self,
+        // TODO: Figure out what to do with this field.
         _from_state: &super::chain::ChainState,
```

Comment:
This is used to ensure we have a correct "starting point" for adding to the note commitment trees in https://github.com/zcash/librustzcash/blob/main/zcash_client_sqlite/src/lib.rs#L928-L934 and https://github.com/zcash/librustzcash/blob/main/zcash_client_sqlite/src/lib.rs#L977-L983

Reply:
Thanks, i added that at https://github.com/zcash/librustzcash/pull/1298/commits/a4323c2d60d6696ad867bed63183206a1a874c47